Public/Src/Cache/ContentStore/Distributed/Stores/DistributedContentCopier.cs

```diff
@@ -169,17 +183,6 @@ protected override Task<BoolResult> ShutdownCoreAsync(OperationContext context)
                         // This is the last attempt, no need to wait any more.
                         break;
                     }
-
-                    long waitTicks = _retryIntervals[attemptCount].Ticks;
-
-                    // Randomize the wait delay to `[0.5 * delay, 1.5 * delay)`
-                    TimeSpan waitDelay = TimeSpan.FromTicks((long)((waitTicks / 2) + (waitTicks * ThreadSafeRandom.Generator.NextDouble())));
-
-                    Tracer.Warning(operationContext, $"{AttemptTracePrefix(attemptCount)} All replicas {hashInfo.Locations.Count} failed. Retrying for hash {hashInfo.ContentHash.ToShortString()} in {waitDelay.TotalMilliseconds}ms...");
-
-                    attemptCount++;
-
-                    await Task.Delay(waitDelay, cts);

```

Comment:
if (attemptCount != _retryIntervals.Count) {
  await Task.Delay();
}

Reply:
And the rest stays the same

---
In reply to: [314512295](https://github.com/microsoft/BuildXL/pull/743#discussion_r314512295) [](ancestors = 314512295)